openedx/core/djangoapps/content/course_structures/models.py

```diff
@@ -21,6 +21,9 @@ class CourseStructure(TimeStampedModel):
     # we'd have to be careful about caching.
     structure_json = CompressedTextField(verbose_name='Structure JSON', blank=True, null=True)
 
+    # JSON mapping of discussion ids to usage keys for the corresponding discussion modules
+    discussion_id_map_json = CompressedTextField(verbose_name='Discussion ID Map JSON', blank=True, null=True)
```

Comment:
unless this field is frequently-rewritten I would suggest using a [JSONField](https://github.com/bradjasper/django-jsonfield/) instead (already a [requirement](https://github.com/edx/edx-platform/blob/bfe01605b4c5a6b3ebe826261ae8c6b783d75d29/requirements/edx/base.txt#L159)), and removing the manual deserialization.  One less worry...